server/rest/src/main/java/org/infinispan/rest/resources/CacheManagerResource.java

```diff
@@ -195,21 +207,27 @@ public Invocations getInvocations() {
       NettyRestResponse.Builder responseBuilder = checkCacheManager(request);
       if (responseBuilder.getHttpStatus() == NOT_FOUND) return completedFuture(responseBuilder.build());
 
-      Map<String, HealthStatus> cachesHeath = new HashMap<>();
+      EmbeddedCacheManager subjectCacheManager = cacheManager.withSubject(request.getSubject());
+      Map<String, HealthStatus> cachesHealth = new HashMap<>();
+      // Remove internal caches
+      Set<String> cacheNames = new HashSet<>(subjectCacheManager.getCacheNames());
```

Comment:
This is changing the return payload of the API, IMO we should keep returning all caches and add a new request parameter to filter internal caches

Reply:
this is the experimental endpoint used in the console. maybe we can add a filter to include all the caches but not get the internals by default